### PR TITLE
[RAINCATCH-861] add crudl api using webapi

### PR DESCRIPTION
**Pull request by JameelB**
# What
Add a new api using webapi for upskilling on typescript.

# JIRA
https://issues.jboss.org/browse/RAINCATCH-861

### CODE REVIEW

File: apps/cloud/src/api/cat.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+const config: WebApiConfig = {
+  limits: true
+};
+
+const hosico: Cat = {
+  id: generateID(),
+  name: 'Hosico',
+  img: 'http://hosicocat.com/uploads/s/k/f/c/kfczm7nn5wgl/img/full_HRz64eK1.jpg'
```
Comment (paolobueno) [inline apps/cloud/src/api/cat.ts:23]: Have you ever seen https://placekitten.com/ ? :smile_cat:

Comment (nialldonnellyfh) [reply apps/cloud/src/api/cat.ts:23]: @paolobueno BEST PLACEHOLDERS EVER!!!

File: apps/cloud/src/api/cat.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+};
+
+const nala: Cat = {
+  id: generateID(),
+  name: 'Nala',
```
Comment (tomjackman) [inline apps/cloud/src/api/cat.ts:28]: Should this not be Nyan? [1]

[1] https://en.m.wikipedia.org/wiki/Nyan_Cat

**Review by paolobueno — APPROVED** [review]

**Review by witmicko — APPROVED** [review]
http://alexhowe.com/wp-content/uploads/2016/04/Ginger-Laptop-V4.jpg

**Pull request merged**